examples/platform/esp32/common/InitServerHelpers.h

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/server/AppDelegate.h>
+#include <stdint.h>
+
+class InitServerHelper
```

Comment:
This looks like it should be a namespace instead: it is just something to enclose a public static method.

Please find a name that does not rely on naming of helper/util/support and similar since they are vague.

`namespace Esp32AppServer { void Init(....); }` could be a choice. File name would be `Esp32AppServer.h/cpp`

Reply:
Added to namespace `Esp32AppServer` and files name renamed `Esp32AppServer.h/cpp`